Directory.Build.props

```diff
@@ -402,7 +408,7 @@
   </PropertyGroup>
 
   <!-- Import it at the end of the props file to override the OutputPath for reference assemblies and use common directory props -->
-  <Import Project="$(MSBuildThisFileDirectory)eng/ReferenceAssemblies.props" />
+  <Import Project="$(MSBuildThisFileDirectory)eng\ReferenceAssemblies.props" />
```

Comment:
We usually use `/` instead of `\` in buildtools and corefx.

Reply:
I got tricked by the arcade SDK where everything is `\` https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/RepoLayout.props#L44

Will update.

Reply:
For file paths MSBuild does the right thing now to normalize these on the specific OS so it doesn't matter anymore. It used to matter in the past before MSBuild normalized these.

Reply:
Makes sense but let's keep consistent in corefx please.